### ISSUE TITLE
Distance: Add quantitative distanace to dynamic-state-of-sim & update the phrase with qualitative distance when distance is not checked.

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#100]
I just noticed that we are missing the quantitative distance information from the dynamic state-of-the-sim list. See bolded text at the end of the second bullet:

> Currently, force on mass 1 by mass 2 is equal in size and opposite in direction of force on mass 2 by mass 1.
> - Force arrows are {{tiny}} {{at 33.4 newtons}}.
> - Mass 1 and mass 2 are {{not so close to}} each other **{{, centers exactly {{4 kilometers}} apart}}**.
> - If Distance not checked: Centers of mass 1 and mass 2 are {{not so close to}} each other.
> - Mass of mass 1 is {{2}} billion kilograms, {{much much smaller than}} mass 2 at {{4}} billion kilograms}}.
> - Robots {{pull very little}} keeping spheres in place.

The second bullet is dynamic should always have qualitative distance and when Distance is checked it should also include quantitative distance in kilometers. Please use the following phrases depending on whether or not Distance is checked:

**When Distance is checked:**
- Mass 1 and mass 2 are {{not so close to}} each other {{, centers exactly {{4 kilometers}} apart}}.

**And when Distance is not checked,** please we update the phrase by adding "Centers of" at the beginning. I think this will be good to reinforce concept that distance is measured from center to center even when distance is unchecked.
- Centers of mass 1 and mass 2 are {{not so close to}} each other.

**Comment by terracoda** [comment]
@zepumph, I just noticed in the A11y View that the prepositions, "to" and "from" are missing  in that second bullet that provides the qualitative distance descriptions.

Would it stream-line things if we just put the propositions in as part of the region name?

**Comment by terracoda** [comment]
@zepumph, there are 3 issues all related to the dynamic distance phrases in the PDOM of BASICS: 
- this one issue #100 - dynamic state of the sim list item 2
- #101 - the dynamic distance phrase in front of position help text
- and I re-opened #94 about the Mass Spheres' dynamic description list

I'm including a screen of A11y View as to where these issues point to.

**When Distance is checked, the PDOM should look like this:**
<img width="936" alt="pdom-distance-checked" src="https://user-images.githubusercontent.com/1902116/53155015-6ec46800-35bc-11e9-9e62-4e4dd639877e.png">

**When Distance is NOT checked, the PDOM should look like this (@terracoda  to add corrected visual):**

**Comment by terracoda** [comment]
@zepumph, the titles of each of the related issues has "Distance" at the front.

**Comment by terracoda** [comment]
@zepumph, with the slightly changed phrasing I think I can leave "from each other" off the comparative descriptions in the intro and the help text. That might it all much more stream-lined.

**Comment by terracoda** [comment]
@zepumph, please note that once the new ranges for described qualitative distance are implemented in #88, the described distance for a distance of 4 kilometers will be "close to" rather than "not so close to".

**Comment by terracoda** [comment]
The screen shots in this issue (#100) link together three issues: this one (#100), #94, and #101.

**Comment by terracoda** [comment]
In https://github.com/phetsims/gravity-force-lab-basics/issues/88#issuecomment-467263264, @zepumph shared two sets of distance strings:
- RELATIVE_DISTANCE_STRINGS 
- DISTANCE_STRINGS

Assuming they are literal representations of the strings, I think we need the full strings, the ones including the preposition "to" or "from" when describing the qualitative distance in the actual PDOM. That is to say, in the scene summary, in the mass sphere descriptions and in the sentence before the help text. All these instances need a preposition.

**Comment by zepumph** [comment]
In the above commits I fixed the distance/position related screen summary bullet, I.E.
![image](https://user-images.githubusercontent.com/6856943/53445812-e67c0380-39bd-11e9-9c5b-ec45a9d52803.png)

**Comment by terracoda** [comment]
Awesome. Distance in help text looks good in both views. Closing.

**Comment by terracoda** [comment]
Oops verified incorrect change.

**Comment by terracoda** [comment]
Awesome. The the second bullet in the state of the sim describing distance between centers looks good in both views. Closing.
